UnitsNet.Tests/CustomCode/TemperatureTests.cs

```diff
@@ -25,6 +25,8 @@ public class TemperatureTests : TemperatureTestsBase
 
         protected override double KelvinsInOneKelvin => 1;
 
+        protected override double SolarTemperaturesInOneKelvin => 1.73070266528210E-04;
```

Comment:
Is solar temperature really a unit? I can't find much references to that when googling.
Is it perhaps only used as a constant?

If it's a unit then it should be something you convert to/from and represent values as `2.5 solar temperatures` etc. Again, I'm only asking since I don't know this domain, you know best.

Reply:
It's not used a lot, but it's the same principle as solar mass and solar luminosity, ratio to solar temperature. They usually use the absolute temperature in Kelvins though. It's just convenient in some scenarios to have the same kind of ratio to temperature as well.